storefront/styles/globalStyles.ts

```diff
@@ -23,6 +23,10 @@ const GlobalStyle = createGlobalStyle`
     font-family: 'Raleway'
   }
 
+  .NavHeading__StyledH4-sc-127f2tf-0:hover, button:hover, .DeleteCartItem:hover{
```

Comment:
Are you sure for `NavHeading__StyledH4-sc-127f2tf-0`? Looks like a dynamic style generated by styled components so the class name will always change.

Reply:
Please fix that :angel: